.github/workflows/deploy.yml

```diff
@@ -41,19 +43,18 @@ jobs:
         with:
           python-version: "3.12"
       - name: Install dependencies
-        run: python -m pip install .[docs] --upgrade pip
+        run: |
+          python -m pip install .[docs] --upgrade pip
+          pip install sphinx sphinx-rtd-theme
       - name: Generate API docs & Build sphinx documentation
         run: |
-          git fetch --tags
-          cd docs
-          python build.py
-          cd ..
+          sphinx-build -M html docs docs-out
```

Comment:
The original [`build.py`](https://github.com/ihmeuw-msca/pypkg/blob/main/docs/build.py) contain methods to build documentations for multiple versions. And we probably still want to keep that.

Reply:
Within the `build.py` we can switch to use `sphinx-build` instead of `make` from Makefile.

Reply:
I don't know why but I couldn't find   build.py last week, now I see it. I will switch back to that and retest

Reply:
And also update OneMod to match

Reply:
Thanks @geoffreyphipps-uw!